routemodel/RouteModel.py

```diff
@@ -61,3 +61,13 @@ def save_csv(self, filename="routedata"):
         data = data.fillna("")
         format_filename = f"{filename}.csv" if ".csv" not in filename else filename
         data.to_csv(format_filename)
+
+    def location_service_csv(self):
+        data = pd.DataFrame(
+            {
+                "coordinates": self.coordinates,
+                "latitude": self.latitudes,
+                "longitude": self.longitudes,
+            }
+        ).fillna("")
+        data.to_csv("location_service.csv")
```

Comment:
nitpick: could we add the index_label = 'id' parameter to this to_csv func? Just in case we use for future scripts that index is consistent for the db schema

Reply:
done